templates/Simple/style.css

```diff
@@ -224,8 +224,9 @@ table.notes-totals {
 }
 
 /* item meta formatting */
-.item-meta {
-	margin: 8px 0 0 0;
+.item-meta,
+.meta {
```

Comment:
Could we have meta items without the `item-meta` class?

Reply:
Can you explain better?

Reply:
Sure, sorry.
We already have the `.item-meta` selector, and you also added the `.meta`, which seems repetitive. I thought maybe we could have meta items that don't have the `.item-meta` class and only have `.meta`, so you added this selector also to support them.